TESTING.asciidoc

```diff
@@ -290,7 +290,16 @@ The REST tests are run automatically when executing the maven test command. To r
 REST tests use the following command:
```

Comment:
The line immediately above this refers to running "maven test". Can you reword this to refer to running "gradle check"?